### DISCUSSION TITLE
Fibaro Smoke stuck on Smoke detected

### DISCUSSION THREAD

**Discussion by thesebastianf** [zwave-js/zwave-js#4753] [Q&A]
Hi
I have a Fibaro Smoke Detector.
It did start this morning with a few false Alarms...

I removed the battery and reinserted it.
Now it is silent but still Shows Smoke detected on the zwavejs UI.

I did wake it Up a Couple Times but despite I See it updating other values the Smoke detected stays.

Any hints?

**Comment by robertsLando** [comment]
@AlCalzone I cannot move the discussion to zwave-js, I think it's a GH bug

**Comment by AlCalzone** [comment]
@thesebastianf you could try re-interviewing it, but I'm not sure that will help. Smoke alarms "push" changes to the controller and theres usually no way to query the state. By pulling the battery you probably made it forget that it has to send the idle notification.

**Reply by thesebastianf** [reply]
hey triggering the reinterview caused the smoke sensor to start the alarm again.

or it was pure coincidence that a false alarm was caused within a second after reinterviewing and waking it up.
however this made the detector to also send an idle afterwards.

I used them now a couple of years with openhab and very recently moved to zwavejs.
Is there a possibility that zwavejs resetted setting like sensitivity when the detector was interviewd the first time? I ask because I havent seen false alarm a very long time now (it was also a pain with those fibaros at the start but I got it under control with low sesitivity...)

overall ... I remember why I hated them :D:D:D

**Reply by AlCalzone** [reply]
You cannot trigger an alarm on a smoke sensor via Z-Wave.

Whats more likely is that your batteries are going bad and the power draw while being awake for a longer time causes the sensor to warn you of low batteries.

Z-Wave JS doesn't reconfigure your devices, aside from making sure they have a proper lifeline to the controller.

**Reply by thesebastianf** [reply]
no I could see it really was the smoke notification channel and not low batt or something else.
![image](https://user-images.githubusercontent.com/13987575/176144258-e22d081b-aeae-4cfb-808f-afd012a1a44a.png)

It just felt strange that it did start when startging the reinterview. I know the only flirs detector is the popp one where you can trigger an alarm from the controller. 

I thought maybe the "re interview" and wake up I did -  triggered some logic on the device itself and it "rediscovered" its own "smoke detected" status was still true and caused it. No clue ... 
I do like with the other fibaros I tossed years ago .. a second false alarm the next days and I just dismiss it and look for another device with less flaws :D

**Reply by AlCalzone** [reply]
Or the sudden power draw causes a fluke in the sensor. Who knows...

**Comment by thesebastianf** [comment]
btw since I have famous @AlCalzone looking in this thread... is there something you could do to push 
https://community.home-assistant.io/t/zwave-js-request-to-invert-covers/303156 at NabuCasa either zwave or core integtration?

Inverting those via template sensors is really a pain as value_template and position_template cannot been set concurrently in a template cover, so HA assumed the cover is open, when position is 100 and cover is closed when position is 0.
sorry for hijacking ... but there a lot of threads around that topic.

**Comment by czesiu20** [comment]
I noticed the same issue with my Heiman HS1SA-Z Smoke detector. Refreshing and waking up has no effect. Only re-interview allows to restore the idle state.

Additionally strange water sensor appeared:
![obraz](https://user-images.githubusercontent.com/15181038/176258483-3de59537-2d12-4d25-b013-6748df439f6a.png)
of which is not available and cannot be removed:
![obraz](https://user-images.githubusercontent.com/15181038/176258832-5bf6a194-c6dd-4056-ad82-9cf1cb0f0d22.png)

**Comment by czesiu20** [comment]
After re-interview in Z-Wave JS to MQTT I can see that the device is in the Idle state:
![obraz](https://user-images.githubusercontent.com/15181038/176261588-58751bbf-69c5-4e07-83e6-e985db8cca01.png)

but in the HA interface i can see Smoke detected:
![obraz](https://user-images.githubusercontent.com/15181038/176262792-29655957-0537-46fe-8f27-b07fda3df1bc.png)

The zwave_js.refresh_value service has no effect.

**Reply by AlCalzone** [reply]
Sounds like something for https://github.com/home-assistant/core

**Comment by thecobra666** [comment]
I've also had this. The solution was very easy. Trigger the smoke sensor and let it reset.

**Reply by czesiu20** [reply]
Great advice. Especially when the reset takes two days and you wonder the house is still there, because u are on vacation. Definitely this is not the case you mentioned. Fortunately, only the device's status is obsolete. The device is working properly, activating sirene and opening the blinds in the case smoke is detected.

**Comment by sanderlv** [comment]
Same here, had a false alarm and removed the batteries, when reinserting the batteries now stuck:
![image](https://user-images.githubusercontent.com/16528916/198142427-035615b4-fb3b-4994-8f2c-5af5b51bb74f.png)


Any clue on how to get it going again?

**Reply by thesebastianf** [reply]
The answer is in the Second Post above.
Reinterview
